sherlock/internal/config/config.go

```diff
@@ -84,16 +85,24 @@ func configureLogging(infoMessages ...string) {
 	var logBuilder zerolog.Context
 	if Config.String("mode") == "debug" {
 		// Colored text for CLI
-		logBuilder = log.Output(zerolog.ConsoleWriter{Out: os.Stderr}).With()
+		output := zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.Kitchen}
+		output.FormatCaller = func(i interface{}) string {
+			if i == nil {
+				return ""
+			} else {
+				return fmt.Sprintf("%-50s |", strings.TrimPrefix(fmt.Sprintf("%s", i), "/build/sherlock/internal"))
+			}
+		}
```

Comment:
Curious what this is for?

Reply:
You can set `log.caller: true` in the config and then this function formats out the call-site of the log message:

![Screenshot 2023-10-27 at 4 09 26 PM](https://github.com/broadinstitute/sherlock/assets/29168264/84d3479d-39f7-489a-9645-a1f44d85e181)

Added a comment to the function to explain that in-line